Packs/CortexXDR/Scripts/XDRCreateDashboardData/XDRCreateDashboardData.py

```diff
@@ -0,0 +1,122 @@
+        incidents = json.loads(incidents_res[0].get('Contents'))
+    except json.JSONDecodeError:
+        demisto.error(f'Failed to parse incidents response from GetIncidentsByQuery, result = {incidents_res}')
+        pass
+    return incidents
+
+
+def update_xdr_list(context: list, xdr_list: list, query_type: str, created_date: str):
+    """
+    Updated the given xdr list with the values from the incident context.
+    :param context: List of values from the incident context key.
+    :param xdr_list: The data list to update.
+    :param query_type: The item to gets the result in the incident(users/hosts/alarms/e.g).
+    :param created_date: The incident creation time.
+    """
+    tmp_dict = {}  # type:dict
+    for val in context:
+        if not val:
+            continue
+
+        if 'This alert from content' in val:
+            continue
+
+        try:
+            if query_type == 'Users' and val.partition('\\')[2]:
+                val = val.partition('\\')[2]
+
+            if query_type == 'Hosts':
+                val = val.partition(':')[0]
+        except Exception as err:
+            demisto.error(f'Could not parse value: {val}, error: {err}')
+            pass
+        if not val:
+            continue
+
+        val = val.capitalize()
+
+        if tmp_dict.get(val):
+            tmp_dict[val] = tmp_dict[val] + 1
+        else:
+            tmp_dict[val] = 1
+
+    if created_date and tmp_dict:
+        xdr_list.append({'created': created_date, 'data': tmp_dict})
+
+
+def main():
+    try:
+        date_pattern = re.compile('\d{4}[-/]\d{2}[-/]\d{2}T\d{2}:\d{2}:\d{2}')
+        include_closed_incidents = argToBoolean(demisto.getArg('includeClosedIncidents'))
+
+        # initialize xdr data lists
+        xdr_data_lists = {}  # type:dict
+        for key in QUERY_TYPE_TO_KEY.keys():
+            xdr_data_lists[key] = []
+
+        # get the xdr incidents
+        incidents = get_xdr_incidents(include_closed_incidents)
+
+        for incident in incidents:
+            context = incident.get('context')
+            # gets the created time of the incidents in format %Y-%m-%dT%H:%M:%S:%f
+            created_date = incident.get('created')
+            created_date = date_pattern.findall(created_date)[0]
+
+            # update all the lists for each incident
+            for key, val in QUERY_TYPE_TO_KEY.items():
+                key_values = demisto.dt(context, val)  # type:list
+                if key_values:
+                    update_xdr_list(key_values, xdr_data_lists[key], key, created_date)
+
+        # create demisto list for each list item
+        for key in QUERY_TYPE_TO_KEY.keys():
+            list_name = f'xdrIncidents_{key}'
+            res = demisto.executeCommand('createList', {'listName': list_name, 'listData': xdr_data_lists[key]})
+            if isError(res):
+                return_error(f'Error occurred while trying to create the list {list_name}: {get_error(res)}')
+        return_results('Done')
```

Comment:
I think that we should have a better output - what lists were created for example.

Reply:
![image](https://user-images.githubusercontent.com/46249224/107220068-97003800-6a1a-11eb-84a8-f12d137d71ee.png)

Reply:
I think it would be better as a markdown table.